Sming/SmingCore/Network/HttpRequest.cpp

```diff
@@ -28,19 +29,25 @@ HttpRequest::~HttpRequest()
 	delete requestPostParameters;
 	delete cookies;
 	postDataProcessed = 0;
+	if (bodyBuf != NULL)
+	{
+		os_free(bodyBuf);
+	}
 }
 
-String HttpRequest::getQueryParameter(String parameterName, String defaultValue /* = "" */)
+String HttpRequest::getQueryParameter(String parameterName,
+		String defaultValue /* = "" */)
 {
 	if (requestGetParameters && requestGetParameters->contains(parameterName))
-			return (*requestGetParameters)[parameterName];
+		return (*requestGetParameters)[parameterName];
 
 	return defaultValue;
 }
-String HttpRequest::getPostParameter(String parameterName, String defaultValue /* = "" */)
+String HttpRequest::getPostParameter(String parameterName,
```

Comment:
Don't like that short lines, it's harder to read. No reason in actual monitors to fit in 80 chars.
Very long lines should be splited, of course.